.repo-rename.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Please toss this one to `./experiments` or delete it.  There's no reason to keep it in he main repository, or to even put it in a shell script.  Also, for shell scripts, best to stick to bash or zsh.  sh lacks features (and yes, some systems offer a symlink from sh -> bash but we can't assume that).

Reply:
I removed the script and deleted it. The PR will reflect.